spec/unit/entities/service_spec.rb

```diff
@@ -68,6 +70,73 @@
     end
   end
 
+  context 'Service.find' do
+    let(:system_name) { random_lowercase_name }
+    let(:service_info) { { remote: remote, ref: system_name } }
+
+    it 'an exception is raised when remote is not configured' do
```

Comment:
I would change the iteration text as: `remote call raises unexpected error`

Reply:
:+1: